addon-test-support/setup-middleware-reporter.ts

```diff
@@ -23,17 +23,26 @@ export const TEST_SUITE_RESULTS: AxeTestResult[] = [];
 
 let currentTestResult: AxeTestResult | undefined = undefined;
 let currentViolationsMap: Map<string, Result> | undefined = undefined;
-let currentUrls = new Set<string>();
-let currentRouteNames = new Set<string>();
+let currentUrls: Set<string> | undefined;
+let currentRouteNames: Set<string> | undefined;
+let currentTestHelpers: Set<string> | undefined;
 
+/**
+ * A custom reporter that is invoke once per failed a11yAudit call. This can be called
+ * multiple times per test, and the results are accumulated until testDone.
+ *
+ * @param axeResults The axe results for each a11yAudit.
+ * @returns Early returns if no violations are found.
+ */
 export async function middlewareReporter(axeResults: AxeResults) {
   if (axeResults.violations.length === 0) {
     return;
   }
 
+  let testMetaData = getTestMetadata(getContext());
```

Comment:
We should stash this off into the `currentTestResult`, then in `pushTestResult` use the metadata to grab the `usedHelpers`.

Reply:
👍